stock_picking_force_assign/security/stock_picking_force_assign_security.xml

```diff
@@ -0,0 +1,11 @@
+<odoo>
+    <record id="group_stock_force_assign" model="res.groups">
+        <field name="name">Auto unreserve pickings</field>
+        <field name="category_id" ref="base.module_category_warehouse_management"/>
```

Comment:
Does this not break the warehouse manager -> warehouse user hierarchy, preventing those groups to show up as a selection field on the user tab?

Reply:
yes, moved this to extra rights then